packer/bsd/openbsd_vanilla.pkr.hcl

```diff
@@ -0,0 +1,88 @@
+variable "bucket" { type = string }
+variable "gcp_project" { type = string }
+variable "image_date" { type = string }
+variable "image_name" { type = string }
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+source "qemu" "qemu-gce-builder" {
+
+  boot_command            = [
+    "S<enter><wait>",
+    "cat <<EOF >>install.conf<enter>",
+    "System hostname = openbsd70<enter>",
+    "Password for root = packer<enter>",
+    "Setup a user = packer<enter>",
```

Comment:
Is this user / password later deleted? We can't have that still be active in GCP...